api/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -156,15 +156,16 @@ public String toString() {
   private int recursionCount;
 
   /**
-   * Handles newly resolved server groups and metadata attributes from name resolution system.
-   * {@code servers} contained in {@link EquivalentAddressGroup} should be considered equivalent
-   * but may be flattened into a single list if needed.
+   * Handles newly resolved addresses and metadata attributes from name resolution system.
+   * {@link EquivalentAddressGroup} addresses should be considered equivalent but may be flattened
+   * into a single list if needed.
    *
-   * <p>Implementations should not modify the given {@code servers}.
+   * <p>Implementations should not modify the given {@code resolvedAddresses}.
```

Comment:
Delete this line. `servers` had been a collection. `ResolvedAddresses` doesn't let itself be modified.

Reply:
Deleted the same